layouts/auto_generated_product.tpl

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+
+<html class="{% if editmode %}editmode{% else %}public{% endif %}" lang="{{ page.language_code }}">
+<head prefix="og: http://ogp.me/ns#">
+  {% include "template-variables" %}
+  {% include "edicy-tools-variables" %}
+  {% include "html-head" product_page: true %}
+</head>
+
+{%- capture bottom_content_html -%}
+  {%- unless editmode -%}
+    {%- content bind=product name="content" -%}
+  {%- endunless -%}
+{%- endcapture -%}
+
+{%- assign bottom_content_size = bottom_content_html | strip | size -%}
+
+{%- capture gallery_content_html -%}
+  {%- unless editmode -%}
+    {%- content bind=product name="gallery" -%}
+  {%- endunless -%}
+{%- endcapture -%}
+
+{%- assign gallery_content_size = gallery_content_html | strip | size -%}
+
+<body class="common-page item-page js-body global-background-color {{ background_type }}-background{% if site.search.enabled %} search-enabled{% endif %}" {% if background_color != '' or editmode %}{{ background_color_style}}{% endif %}>
+  {% include "header" %}
+  {% include "template-svg-spritesheet" %}
+
+  <div class="container">
+
+    <main class="content content-formatted" role="main">
+      {% include "menu-breadcrumbs-sd" %}
+
+      <div class="items-body product-content">
+        <div class="flex-col">
+          <div class="content-illustrations">
+
+            {%- if product.image != blank %}
+              {% assign item_image_state = "with-image" %}
+            {% else %}
+              {% assign item_image_state = "without-image" %}
+            {% endif -%}
+
+            <div class="content-item-box {{ item_image_state }} js-content-item-box" href="{{ page.url }}">
+              <div class="item-top">
+                {%- if product.image != blank -%}
+                  <div class="top-inner aspect-ratio-inner product-page">
+                    {%- assign image_class = "item-image not-cropped" -%}
+                    {% image product.image target_width: "1280" class: image_class loading: "lazy" %}
+                  </div>
+                {%- endif -%}
+              </div>
+            </div>
+
+            {%- if gallery_content_size > 0 or editmode -%}
+              <div class="content-gallery content-area js-product-gallery" data-search-indexing-allowed="true">
+                {% content bind=product name="gallery" %}
+              </div>
+            {%- endif -%}
+
+          </div>
+        </div>
+
+        <div class="flex-col">
+          <div class="content-body js-product-content-right">
```

Comment:
The product right content is not sticky on scroll. The style rule checks for `product-content-right` class, so let's re-add that
```suggestion
          <div class="content-body js-product-content-right product-content-right">
```